src/EnergyPlus/SystemReports.cc

```diff
@@ -3920,6 +3920,11 @@ namespace SystemReports {
             SOLARCOLLECTOR_FLATPLATE_PHOTOVOLTAICTHERMAL,
             SOLARCOLLECTOR_UNGLAZEDTRANSPIRED,
             ZONEHVAC_AIRDISTRIBUTIONUNIT,
+            ZONEHVAC_TERMINALUNIT_VRF,
```

Comment:
Admit confusion here as to which of these needs to be added to System Reports. Certainly they were missing.